src/git-source-provider.ts

```diff
@@ -216,6 +217,14 @@ export async function getSource(settings: IGitSourceSettings): Promise<void> {
       settings.ref,
       settings.commit
     )
+
+    // Set default author
+    if (!await git.configExists('user.name', true)) {
+      await git.config('user.name', github.context.workflow, true)
+    }
+    if (!await git.configExists('user.email', true)) {
+      await git.config('user.email', 'github-actions@github.com', true)
```

Comment:
There's already a bot account for GitHub Actions. It seems like it's a decent default.

https://github.com/actions/checkout/blob/692973e3d937129bcbf40652eb9f2f61becf3332/README.md?plain=1#L283-L284

```suggestion
      await git.config('user.name', 'github-actions[bot]', true)
    }
    if (!await git.configExists('user.email', true)) {
      await git.config('user.email', '41898282+github-actions[bot]@users.noreply.github.com', true)
```